keras_eval/data_generators.py

```diff
@@ -0,0 +1,328 @@
+                 samplewise_center=False,
+                 featurewise_std_normalization=False,
+                 samplewise_std_normalization=False,
+                 zca_whitening=False,
+                 zca_epsilon=1e-6,
+                 rotation_range=0.,
+                 width_shift_range=0.,
+                 height_shift_range=0.,
+                 brightness_range=None,
+                 shear_range=0.,
+                 zoom_range=0.,
+                 channel_shift_range=0.,
+                 fill_mode='nearest',
+                 cval=0.,
+                 horizontal_flip=False,
+                 vertical_flip=False,
+                 rescale=None,
+                 preprocessing_function=None,
+                 data_format='channels_last',
+                 validation_split=0.0,
+                 ):
+        self.as_super = super(AugmentedImageDataGenerator, self)
+        self.as_super.__init__(featurewise_center=featurewise_center,
+                               samplewise_center=samplewise_center,
+                               featurewise_std_normalization=featurewise_std_normalization,
+                               samplewise_std_normalization=samplewise_std_normalization,
+                               zca_whitening=zca_whitening,
+                               zca_epsilon=zca_epsilon,
+                               rotation_range=rotation_range,
+                               width_shift_range=width_shift_range,
+                               height_shift_range=height_shift_range,
+                               brightness_range=brightness_range,
+                               shear_range=shear_range,
+                               zoom_range=zoom_range,
+                               channel_shift_range=channel_shift_range,
+                               fill_mode=fill_mode,
+                               cval=cval,
+                               horizontal_flip=horizontal_flip,
+                               vertical_flip=vertical_flip,
+                               rescale=rescale,
+                               preprocessing_function=preprocessing_function,
+                               data_format=data_format,
+                               validation_split=validation_split,
+                               )
+
+        self.data_augmentation = data_augmentation
+
+    def flow_from_directory(self, directory,
+                            target_size=(256, 256), color_mode='rgb',
+                            classes=None, class_mode='categorical',
+                            batch_size=32, shuffle=True, seed=None,
+                            save_to_dir=None,
+                            save_prefix='',
+                            save_format='png',
+                            follow_links=False,
+                            subset=None,
+                            interpolation='nearest',
+                            ):
+        return AugmentedDirectoryIterator(
+            directory, self,
```

Comment:
Correct me if I'm wrong, but this should be:
```suggestion
            directory=directory,
```

Reply:
def __init__(self, directory, **image_data_generator**, data_augmentation,
                 target_size=(299, 299), color_mode='rgb',
                 classes=None, class_mode='categorical',
                 batch_size=32, shuffle=True, seed=None,
                 data_format='channels_last',
                 save_to_dir=None, save_prefix='', save_format='png',
                 follow_links=False,
                 subset=None,
                 interpolation='nearest',
                 )

Reply:
The code is correct because you are passing the image_data_generator to the iterator.